frontend/requirements.txt

```diff
@@ -67,8 +67,7 @@ responses==0.12.1
 SQLAlchemy==1.3.23
 
 # A common package that holds the models deifnition and schemas that are used
-# accross different amundsen repositories.
-amundsen-common==0.9.0
+amundsen-common>=0.9.0,<1.0
```

Comment:
maybe it's woth to keep the deps 1:1 with other services? metadata already expects 0.11.0

Reply:
https://github.com/amundsen-io/amundsen/pull/1088